modules/subchannel/doc/content/modules/subchannel/sqa/subchannel_sdd.md

```diff
@@ -0,0 +1,27 @@
+!template load file=sqa/module_sdd.md.template category=subchannel module=Porous Flow
+
+!template! item key=introduction
+The [!ac](MOOSE) {{module}} module is based on the MOOSE framework and thus inherits
+the unique features and base characteristics of the framework, as outlined in the [framework_sdd.md].
+Specific details unique to the module are outlined in this document.
+!template-end!
+
+!template! item key=system-scope
+!include subchannel_srs.md start=system-scope-begin end=system-scope-finish
+!template-end!
+
+!template! item key=dependencies-and-limitations
+SQA TO DO
```

Comment:
I would just get this, system design, and system structure out of the way now. There are lots of examples in the other modules for how these need to be done.